source/patterns/@aws-solutions-constructs/aws-apigatewayv2websocket-sqs/README.md

```diff
@@ -0,0 +1,129 @@
+# aws-apigateway-sqs module
```

Comment:
update to `aws-apigatewayv2websocket-sqs` ?

Reply:
updated